### ISSUE TITLE
Documentation for JAX-RS resources is generated twice per method

### ISSUE THREAD

**Issue by xpapad** [stoicflame/enunciate#237]
I'm using Enunciate 2.1.1. I have a JAX-RS service (UsersRestService)  with 3 methods. The first page of the documentation is fine:

![page1](https://cloud.githubusercontent.com/assets/904419/11722866/40c58b2a-9f63-11e5-9f96-512237fbd737.png)

When I click on any resoure, I see the methods twice, eg:

![page2](https://cloud.githubusercontent.com/assets/904419/11722932/a820f200-9f63-11e5-8d78-35356782fb74.png)

I did some debugging, and I saw that the class is added to the FreeMarker model twice.
According to Maven logs:

```
[DEBUG] [ENUNCIATE] Added com.company.saas.csadmin.rest.UsersRestService as a JAX-RS root resource.
[DEBUG] [ENUNCIATE] Added com.company.saas.csadmin.rest.UsersListResponse as a Jackson type definition.
...
[DEBUG] [ENUNCIATE] Added com.company.saas.csadmin.rest.UsersRestService as a JAX-RS root resource.
```

The problem seems to begin at EnunciateAnnotationProcessor:73. The class is added to *apiElements* once as a *rootElement*, and then again as an *includedType*. Subsequent interations of apiElements duplicate the methods in the model.

I'm not sure if this is a bug or there's something wrong in my setup, any suggestions?

**Comment by stoicflame** [comment]
Tracking as a bug. Thanks for the report.

**Comment by stoicflame** [comment]
I don't think this is a problem at EnunciateAnnotationProcessor:73. This might be the same issue as reported at #218. Can you confirm?

**Comment by stoicflame** [comment]
Fixed at b242116.

**Comment by stoicflame** [comment]
[Enunciate 2.2 is now available](https://github.com/stoicflame/enunciate/releases/tag/v2.2.0).
